src/software/simulated_tests/validation_functions/ball_in_rectangle_validation.cpp

```diff
@@ -0,0 +1,11 @@
+#include "software/simulated_tests/validation_functions/friendly_scored_validation.h"
+#include "software/geom/algorithms/contains.h"
+#include "software/geom/rectangle.h"
+
+void ballInRectangle(Rectangle rectangle, std::shared_ptr<World> world_ptr, ValidationCoroutine::push_type& yield)
```

Comment:
can we just pass in the `Ball`?

Reply:
I think we determined we can't because only the world_ptr get's updated, but I removed this function anyways since I don't use it